### DISCUSSION TITLE
Algoritmekader v1.3

### DISCUSSION THREAD

**Discussion by ruthkoole** [MinBZK/Algoritmekader#172] [Announcements]
Een nieuwe versie van het Algoritmekader staat online! Bekijk het [Algoritmekader](https://minbzk.github.io/Algoritmekader/)!

## Wat is er toegevoegd/aangepast sinds de vorige release? 

- Verwerken feedback eerdere releases #131, #135, #152, #136, #147, #71 
- Refinen van bouwblokken (conformiteitsbeoordeling wordt opgenomen in governance, en bias en non-discriminatie is een los bouwblok) #164 
- Governance een centrale plek gegeven + onderverdeling duidelijk gemaakt #156  
- Definities en begrippen toegevoegd #161 
- Instrumentenpagina heeft een update gekregen + instrumenten fundamentele rechten uitgewerkt #153 
- Nieuwe maatregelen toevoegen #149, #158, #166 
- Overzichtspagina's van maatregelen en vereisten zijn gebruiksvriendelijker gemaakt #151 
- Nieuwe vereisten toegevoegd #163, #171

**Full Changelog**: https://github.com/MinBZK/Algoritmekader/compare/v1.2...v1.3

<hr /><em>This discussion was created from the release <a href='https://github.com/MinBZK/Algoritmekader/releases/tag/v1.3'>Algoritmekader v1.3</a>.</em>

**Comment by floort** [comment]
Er is is misgegaan met de acceptatie van #71. Versie 1.3 bevat nog de onjuiste kop en omschrijving met een dode link.
![image](https://github.com/MinBZK/Algoritmekader/assets/140608/00897ff9-d21a-43be-ad17-6710c1c9a879)

**Comment by ruthkoole** [comment]
Links zijn inmiddels gefixt. Was een bug, waardoor ze op sommige pagina's niet werkten. Nu als het goed is wel. Ik zie de vereiste uit #71 hier staan: https://minbzk.github.io/Algoritmekader/vereisten/verantwoordingsplicht_rechtmatigheid/
De vereiste uit jouw screenshot is een andere denk ik?

**Reply by floort** [reply]
Dankjewel. Het lijkt inderdaad een bug te zijn. Het punt van #71 was om die pagina te vervangen. Mogelijk is er een script die oude referenties niet goed verwijderd. Of er is tussen het indienen van #71 en de acceptatie een verandering in de repo doorgevoerd waardoor het vervangen van die pagina is veranderd in een toevoeging van een nieuwe pagina. Commit [68685ba](https://github.com/MinBZK/Algoritmekader/pull/71/commits/68685badd5439d9c5081818a8b8e3c9c0ca75c3a) zou in ieder geval voor die verwijdering van de pagina moeten zorgen.

**Reply by ruthkoole** [reply]
Hoi @floort,

Ik probeer dit nog recht te zetten (beetje laat, maar beter laat dan nooit). Even een check bij jou. Er zijn nu inderdaad twee vereisten die iets zeggen over verantwoordingsplicht:
- https://minbzk.github.io/Algoritmekader/vereisten/verantwoordingsplicht_rechtmatigheid/index.html
- https://minbzk.github.io/Algoritmekader/vereisten/beschrijven_en_toewijzen_van_verantwoordelijkheden_bij_verwerking_persoonsgegevens/

Ik vind de eerste vereiste vollediger, hoewel de tweede aan betere bronvermelding doet. Eens als we de tekst van de eerste behouden, een aanvullen met de bronnen benoemd bij de tweede? De tweede in deze lijst kan dan komen te vervallen.

Ik zal je even taggen in de betreffende PR komende dagen (zitten wat meer aanpassingen in de vereisten in)

**Reply by floort** [reply]
Het lastige van de bronvermelding bij die tweede pagina is dat je eigenlijk twee issues hebt: de verantwoordingsplicht (art. 5 AVG) en de wetten/artikelen waarover je je moet verantwoorden volgens 5(1)(a). Ik denk dat het tweede document probeert ook een onvolledige opsomming van dat tweede te geven. Ik zou geen volledige opsomming kunnen geven voor dat tweede punt dus ik stel voor om in ieder geval niet de indruk te wekken dat de opsomming volledig is. 

Als het toch de bedoeling is om meer relevante wetgeving te benoemen die ik met alle plezier een poging om meer aan te dragen.
